pointers/pointers.c

```diff
@@ -66,7 +101,29 @@ int string_compare(char *m, char *n)
 */
 char *find_string(char *haystack, char *needle)
 {
-
+    int nLength = strlen(needle);
+    int count = 0;
+    char *found = NULL;
+    for (int i = 0; i < strlen(haystack); i++)
+    {
+        if (haystack[i] == needle[0])
+        {
+            found = &haystack[i];
+            ++count;
+            for (int j = 1; j < nLength; j++)
+            {
+                if (haystack[i + j] == needle[j])
+                {
+                    ++count;
```

Comment:
Great work here! This was a tough problem, and your creative use of pointers shows good understanding.